Ryujinx.Ava/UI/ViewModels/TitleUpdateViewModel.cs

```diff
@@ -223,4 +229,24 @@ public async void Add()
 
         SortUpdates();
     }
+
+    public void Save()
+    {
+        _titleUpdateWindowData.Paths.Clear();
+        _titleUpdateWindowData.Selected = "";
+
+        foreach (TitleUpdateModel update in TitleUpdates)
+        {
+            _titleUpdateWindowData.Paths.Add(update.Path);
+
+            if (update == SelectedUpdate)
+            {
+                _titleUpdateWindowData.Selected = update.Path;
+            }
+        }
+
+        using FileStream titleUpdateJsonStream = File.Create(_titleUpdateJsonPath, 4096, FileOptions.WriteThrough);
+
+        titleUpdateJsonStream.Write(Encoding.UTF8.GetBytes(JsonHelper.Serialize(_titleUpdateWindowData, true)));
```

Comment:
```suggestion
        File.WriteAllBytes(_titleUpdateJsonPath, Encoding.UTF8.GetBytes(JsonHelper.Serialize(_titleUpdateWindowData, true)));
```
Could this be shortened like this?

Reply:
The using is here to use `WriteThrough` idk why it was added, maybe to be sure the real data will be written. I just kept the original code as-is.

Reply:
Hmm, it looks fine to me to replace. `WriteThrough` just ensures cache is ignored and stuff is written to the disk directly. `WriteAllBytes` will open the file and close it after everything was written, so the changes should also be written to the disk at that point.

Reply:
That should be fine yeah. Fixed!